src/bin/bootleby.rs

```diff
@@ -115,6 +120,46 @@ fn check_for_override(gpio: &lpc55_pac::GPIO, a_ok: bool, b_ok: bool) -> SlotId
     // below.
     shared_buffer.fill(0);
 
+    // Check for a Bootleby Override Image Select debug authentication beacon.
+    //
+    // Debug authentication beacons are two 16-bit values: one in a debug
+    // credential and the other in a debug authentication response. The LPC55
+    // ROM will only accept a debug authentication response signed by the
+    // private key matching the debug credential embedded inside the response
+    // which itself must be signed by a private key matching one of the secure
+    // boot root public keys recorded in CMPA. Since the debug credential must
+    // be signed by one of the secure boot root keys (which are presumed to be
+    // kept secure and infrequently accessed), the debug credential beacon
+    // embedded in that debug credential can be treated as an RPC index that the
+    // holder of the debug credential's private key is authorized to invoke. The
+    // debug authentication response is signed by the debug credential's private
+    // key which must be available for each debug authentication session and
+    // thus the debug authentication beacon may be changed for each session.
+    // This allows the debug authentication response beacon to be used as an
+    // argument to the RPC.  If the debug auth beacon register contains any
+    // value other than zero, it can only have been set by a successful debug
+    // authentication challenge/response with a signature chain tracing back to
+    // one of the enabled secure boot root keys. If the debug credential beacon
+    // indicates a Bootleby Override Image Select, the debug authentication
+    // response beacon indicates which slot to use.  If this mechanism was used,
```

Comment:
I'd like this broken into paragraphs for easier skimming, because I apparently missed part of it on first read through. I suggest breaking paragraphs at:
- end of line 134 before `The` (topic change from credential back to response)
- Middle of 139 before the word `If` (topic change to register handling)
- Middle of line 142 before the word `If` (explanation of override mechanism and justification for its priority)